packages/block-library/src/gallery/save.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { defaultColumnsNumber } from './shared';
+
+export default function	save( { attributes } ) {
```

Comment:
Looks like there is some extra whitespace here.

Reply:
Fixed :)